vendor/bat-native-ledger/BUILD.gn

```diff
@@ -81,4 +83,9 @@ source_set("ledger") {
     rebase_path("bat-native-tweetnacl:tweetnacl", dep_base),
     rebase_path("bat-native-rapidjson", dep_base),
   ]
+
+  if (brave_ads_enabled && !is_linux) {
+    # TODO(bridiver): Required for all platforms once Brave Ads is building on Linux
```

Comment:
can you please add a ticket for this so we don't forget?

Reply:
Sure